wifi-subsys/components/tools/test/test_cbor.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @brief   Non Volatile Storage
+ *
+ * @author  eduazocar <eduardo@turpialdev.com>
+ */
+#include <stdio.h>
+#include <stdint.h>
+#include "esp_log.h"
+#include "unity.h"
+#include "serialization.h"
+
+/* default namespace */
+// static const char *namespace = "TEST";
+
+/* uint8_t default params */
+static const char message[128] = "Mesh data sended";
+char data_received[128];
+char text[128];
```

Comment:
Please pay attention to these lines, I defined to an array, because when i defined to char * (String). Itś rebooting the boards. I think that is the memory is exceeded. So. If you have a suggest, could test to take anothers solutions